src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts

```diff
@@ -249,9 +252,9 @@ export class CreateEditEventsComponent implements OnInit, OnDestroy {
     });
   }
 
-  private validateSpaces(control: AbstractControl): ValidationErrors {
-    const value = control && control.value && control.value !== control.value.trim();
-    return value ? { hasNoWhiteSpaces: 'false' } : null;
+  private checkFileExtension(file: any): void {
```

Comment:
(file: File)
A hint from StackOverflow:
TypeScript has a defined interface for File which represents the file object in JavaScript. Generally you should look for types named as their Javascript countertypes. You can use this type in a parameter annotation for the parameter

Reply:
@vitaliitrehubov  Thank you for hints, I'll fix them in 5 min